core/vm/actor/cron_actor.hpp

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_VM_ACTOR_CRON_ACTOR_HPP
+#define CPP_FILECOIN_CORE_VM_ACTOR_CRON_ACTOR_HPP
+
+#include "vm/actor/actor.hpp"
+#include "vm/actor/storage_power_actor.hpp"
+#include "vm/vm_context.hpp"
+
+namespace fc::vm::actor {
+
+  struct CronTableEntry {
+    Address to_addr;
+    uint64_t method_num;
+  };
+
+  struct CronActor {
+    // Entries is a set of actors (and corresponding methods) to call during
```

Comment:
Please, use `/** */` for multiline comments

Reply:
Don't copy spec when it is not needed. 
`make it easy to add a handler to Cron elsewhere in the spec code`

Reply:
Fixed